game/outfits.rpy

```diff
@@ -1,289 +1,2058 @@
-default persistent.jn_natsuki_current_pose = "sitting"
-default persistent.jn_natsuki_current_outfit = "uniform"
-default persistent.jn_natsuki_current_hairstyle = "default"
-default persistent.jn_natsuki_current_accessory = "hairbands/red"
-default persistent.jn_natsuki_current_eyewear = None
-default persistent.jn_natsuki_current_headgear = None
-default persistent.jn_natsuki_current_necklace = None
-default persistent.jn_natsuki_saved_outfits = {}
-default persistent.jn_natsuki_auto_outfit_change_enabled = True
-
-init python in jn_outfits:
+
+init -2:
+    default persistent.jn_natsuki_auto_outfit_change_enabled = True
+    default persistent.jn_custom_outfits_unlocked = False
+    default persistent.jn_natsuki_outfit_on_quit = "jn_school_uniform"
+    default persistent.jn_outfit_list = {}
+    default persistent.jn_wearable_list = {}
+
+init -1 python in jn_outfits:
+    from Enum import Enum
+    import json
+    import os
     import random
+    import re
     import store
     import store.jn_affinity as jn_affinity
     import store.jn_utils as jn_utils
+    import time
+
+    # Critical file paths
+    __CUSTOM_WEARABLES_DIRECTORY = os.path.join(renpy.config.basedir, "custom_wearables/").replace("\\", "/")
+    __CUSTOM_OUTFITS_DIRECTORY = os.path.join(renpy.config.basedir, "custom_outfits/").replace("\\", "/")
+    __WEARABLE_BASE_PATH = os.path.join(renpy.config.basedir, "game/mod_assets/natsuki/")
+
+    # Ignore the syntax highlighting - "it just works"
+    __RESTRICTED_CHARACTERS_REGEX = "((\.)|(\[)|(\])|(\})|(\{)|(,)|(\!))"
```

Comment:
Could use raw string (`r""`) to fix the highlighting and reduce number of slashes.